releases/release-7.2.0.md

```diff
@@ -0,0 +1,341 @@
+
+* Improve the performance of statistics collection [#44725](https://github.com/pingcap/tidb/issues/44725) @[xuyifangreeneyes](https://github.com/xuyifangreeneyes) **tw@hfxsd** <!--1352-->
+
+    TiDB v7.2.0 optimizes the statistics collection strategy, skipping some of the duplicate information and information that is of little value to the optimizer. The overall speed of statistics collection has been improved by 30%. This improvement allows TiDB to update the statistics of the database in a more timely manner, making the generated execution plans more accurate, thus improving the overall database performance.
+
+    By default, statistics collection skips the columns of the `json`, `blob`, `mediumblob`, and `longblob` types. You can modify the default behavior by setting the [`tidb_analyze_skip_column_types`](/system-variables.md#tidb_analyze_skip_column_types-new-in-v720) system variable. TiDB supports skipping the `JSON`, `BLOB`, and `TEXT` types and their subtypes.
+
+    For more information, see [documentation](/system-variables.md#tidb_analyze_skip_column_types-new-in-v720).
+
+* Improve the performance of checking data and index consistency [#43693](https://github.com/pingcap/tidb/issues/43693) @[wjhuang2016](https://github.com/wjhuang2016) **tw@qiancai** <!--1436-->
+
+    The [`ADMIN CHECK [TABLE|INDEX]`](/sql-statements/sql-statement-admin-check-table-index.md) statement is used to check the consistency between data in a table and its corresponding indexes. In v7.2.0, TiDB optimizes the method for checking data consistency and improves the execution efficiency of [`ADMIN CHECK [TABLE|INDEX]`](/sql-statements/sql-statement-admin-check-table-index.md) greatly. In scenarios with large amounts of data, this optimization can provide a performance boost of hundreds of times.
+
+    The optimization is enabled by default ([`tidb_enable_fast_table_check`](/system-variables.md#tidb_enable_fast_table_check-new-in-v720) is `ON` by default) to greatly reduce the time required for data consistency checks in large-scale tables and enhance operational efficiency.
+
+    For more information, see [documentation](/system-variables.md#tidb_enable_fast_table_check-new-in-v720).
+
+### Reliability
+
+* Automatically manage queries that consume more resources than expected (experimental) [#43691](https://github.com/pingcap/tidb/issues/43691) @[Connor1996](https://github.com/Connor1996) @[CabinfeverB](https://github.com/CabinfeverB) @[glorv](https://github.com/glorv) @[HuSharp](https://github.com/HuSharp) @[nolouch](https://github.com/nolouch) **tw@hfxsd** <!--1411-->
```

Comment:
@qiancai @Connor1996 in the [related Github issue](https://github.com/pingcap/tidb/issues/43691), While this release note highlights that it's measuring execution time, I think we should highlight that this is total execution time rather than just a single coprocessor execution time (the previous way). That's a big and positive change.

Reply:
ditto

Reply:
@Connor1996 Hmm now I'm confused given your earlier reply to the highlight text above. Can you elaborate a little on that?

Reply:
@samhld Sam, here we introduced a fresh new mechanism to take actions automatically on certain queries by defining rules. That's the point we shall highlight.  "query execution time" is the first rule included, we may append more rules like "resource unit", "cpu time" in the future release.  So we don't have to compare with "coprocessor execution time", as it's different in dimension.